homeassistant/components/growatt_server/const.py

```diff
@@ -8,6 +8,7 @@
 DEFAULT_NAME = "Growatt"
 
 SERVER_URLS = [
+    "https://openapi.growatt.com/",
```

Comment:
This used to be the default, and was previously put into the config entries as well.

Should we migrate existing config entries to update the stored URL from https://server-api.growatt.com/ to https://server.growatt.com/?

../Frenck

Reply:
This has never been the default, it is a new entry.
I believe this comment has been copy/pasted from the other PR.

With regards to migrating existing config entries, I don't really mind, it's not something I've ever done historically for this integration as the only way people have ever been able to reconfigure it is by deleting it and then re-adding it anyway (which is generally the first thing people do when they get problems with it).

Reply:
@frenck I agree that adding this URL without migrating existing entries would be enough for now. That way not all users will switch at once, which probably increases the chance it will quickly be blocked again.